Source/ACE.Entity/AceCharacter.cs

```diff
@@ -0,0 +1,16 @@
+﻿using ACE.Common;
+using MySql.Data.MySqlClient;
+
+namespace ACE.Entity
+{
+    [DbTable("vw_ace_character")]
+    [DbGetList("vw_ace_character", 42, "guid")]
+    public class AceCharacter
```

Comment:
This class has no real purpose.  I think it may be a vestige of your Motion work, as that's the only place it's used, but even then you only fetch it and never use it.